core/modules/node/js/node.types.js

```diff
@@ -12,62 +13,57 @@ Backdrop.behaviors.contentTypes = {
     });
 
     // Publishing settings.
-    $context.find('#edit-workflow').backdropSetSummary(function() {
-      var vals = [];
-      var defaultStatus = $context.find('input[name="status_default"]:checked').parent().text();
-      vals.push(Backdrop.checkPlain($.trim(defaultStatus)));
-      if ($context.find('input[name="sticky_default"]:checked').length) {
-        vals.push(Backdrop.t('Sticky'));
-      }
-      if ($context.find('input[name="promote_default"]:checked').length) {
-        vals.push(Backdrop.t('Promoted'));
-      }
-      return vals.join(', ');
-    });
+    // This is implemented as a function, because it is needed once for the
+    // initial page load, but then again if the "Schedule for later" checkbox
+    // gets unticked.
+    function updatePublishVtabSummary() {
+      $context.find('#edit-workflow').backdropSetSummary(function () {
+        var vals = [];
+        var defaultStatus = $context.find('input[name="status_default"]:checked').parent().find('label').text();
+        vals.push(Backdrop.checkPlain($.trim(defaultStatus)));
+        if ($context.find('input[name="sticky_default"]:checked').length) {
+          vals.push(Backdrop.t('Sticky'));
+        }
+        if ($context.find('input[name="promote_default"]:checked').length) {
+          vals.push(Backdrop.t('Promoted'));
+        }
+        return vals.join(', ');
+      });
+    }
 
-    // Multilingual support.
-    $context.find('#edit-multilingual').backdropSetSummary(function() {
-      var vals = [];
-      var multilingualSupport = $context.find('input[name="language"]:checked').parent().find('label').text();
-      vals.push(Backdrop.checkPlain($.trim(multilingualSupport)));
-      return vals.join(', ');
-    });
+    // We call this once, to update the "Publishing settings" summary initially.
+    updatePublishVtabSummary();
 
     // Uncheck the "Schedule for later" option if scheduling is disabled.
     $context.find('input[name="scheduling_enabled"]').once().on('change', function() {
       var $checkedStatusDefault = $context.find('input[name="status_default"]:checked');
       if ($checkedStatusDefault.val() === '2') {
         $checkedStatusDefault.prop('checked', false);
         $context.find('input[name="status_default"]:first').prop('checked', true);
+        // If the previous selection for "Default status" was "Schedule for
+        // later", then unticking the "Show option for scheduling" checkbox
+        // would still show "Schedule for later" as the vtab summary - even
+        // though the default has been switched to "Published". So we also need
+        // to update the "Publishing settings" summary again.
+        // See https://github.com/backdrop/backdrop-issues/issues/4098
```

Comment:
Don't need all this. Simplify to:
```suggestion
        // Update the "Publishing settings" summary again, since the selected
        // radio option was changed.
```